PLUGINS.md

```diff
@@ -87,6 +88,15 @@ will be available at `node.fooBar`.
 A `plugin.name` can also be an `'object`. This object will be merged
 directly with the
 
+### `plugin.needs` (Array) _optional_
+
+An array of strings which are the names of other plugins that this plugin
```

Comment:
Name the plugin is loaded under? (conn, not ssb-conn)

Reply:
So far whenever we talk about plugin names in this document, it's always `plugin.name` and not the package name.